build/packaging/linux/conf/kend.conf

```diff
@@ -56,3 +56,32 @@ ADDITIONAL=""
 
 DATA_DIR=
 LOG_DIR=$DATA_DIR/logs
+
```

Comment:
@whoisxx @aidan-kwon 
How about adding new package like `kessnd.conf` or `kesd.conf`?

If we add these options here, this options can confuse general KEN user.

Reply:
I agreed. It could be challenge to setup EN at first time. and it's a bit tricky to set default value of KES options

Reply:
you mean adding `kesd, kesd.conf`, right?

Reply:
@whoisxx How about applying changes for `start_docker` in PR and creating `kesd, kesd.conf` in KES repository.

Reply:
if KES use klaytn repo as submodule, it need to apply here. but I think KES could add `ADDITIONAL`, do you think?

Reply:
@whoisxx I thought CD can copy config files in KES repo to wherever it is used. Never mind too much. You can create config/deamon files anywhere you want or use `ADDITIONAL` to add any flags.

Reply:
As we talked, let's remove this config value.